src/Components/SingleArticle.jsx

```diff
@@ -47,7 +54,8 @@ const SingleArticle = () => {
         </div>
         {error && <p>Sorry, article not found !</p>}
       </section>
-      <Comments commentCount={articleById.comment_count} article_id={articleById.article_id}/>
+      <Comments comments={comments} setComments={setComments} commentCount={articleById.comment_count} article_id={articleById.article_id}/>
+      <CommentForm setComments={setComments} article_id={articleById.article_id} updateCommentCount={updateCommentCount}/>
```

Comment:
I would recommend having the comment form above the comments list as this feels more natural as a user experience. If i want to read a couple of comments and then leave my own comment I am currently forced to scroll to the very bottom to write a comment.